src/components/imageGrid.vue

```diff
@@ -55,20 +56,13 @@ export default {
       let copyCount = localStorage.getItem(image.name);
       localStorage.setItem(image.name, ++copyCount);
 
-      const file = await fs.promises.readFile(image.path);
-      const buffer = await Buffer.from(file).toString("base64");
-
-      const blob = this.b64ToBlob(buffer, image.fileType, 512);
-
+      const blob =window.nativeImage.createFromPath(image.path)
```

Comment:
Is there no need for doing the `base64` conversion?

Reply:
yeah, it just works directly using the path of the image